### PR TITLE
Rather a comment

### PR DESCRIPTION
**Pull request by Olf0**
Rectify comment WRT wording and content

Even though its base (branch `obs`) is outdated, something similar needs to be implemented to address [issue \#332](https://github.com/sailfishos-patches/patchmanager/issues/322), so I commented on the logic and its comment [here](https://github.com/nephros/patchmanager/pull/4/files#diff-18d76fe5bfa0bf2a3d62aa6755523b085bd01e1cca9ba75ffdcb43c53822fbd9R337).

P.S.: Only the last two commits are relevant (the latest one fixes the second last), I have no idea what went wrong, when I checked out the `obs` branch and committed a single change.

### CODE REVIEW

**Comment by nephros** [comment]
Hi.

This  'obs' branch is used by me as an integration branch to merge and test various open branches or PRs from the patchmanager repo proper.

I regularly reset it to master, squash-commit any changes (private or pubic) and push to this branch, to then build on OBS.

Therefore, PRs for this branch are pointless as they will be lost in the next iterations of this integration process. If any changes are required, they need to go to the "source" branch of this process.

In short, please just ignore this branch, its contents are volatile.

I'll look into your two "relevant" commits and see where they should go (it looks like that is one or two branches I have locally).

**Comment by Olf0** [comment]
No, sorry, what a misunderstanding: These two commits shall go nowhere electronically.

It is only about [the now even lengthier comment in line 337](https://github.com/nephros/patchmanager/pull/4/files#diff-18d76fe5bfa0bf2a3d62aa6755523b085bd01e1cca9ba75ffdcb43c53822fbd9R337) meant for you read and intended as a hint (probably something you already considered on your own, meanwhile).

**Comment by nephros** [comment]
Ah heh. What a novel way to send a message!

Someone should build an IM application around git!

Thanks I shall read and ponder!

**Comment by nephros** [comment]
merged into https://github.com/nephros/patchmanager/tree/relax-versioncheck2 for posterity.

**Comment by Olf0** [comment]
> Someone should build an IM application around git!

Good idea for a chat software with logging.  Could log either into a public or private repo.  :wink:
Actually, [Delta Chat](https://delta.chat/), an email client disguised as Chat Software is not far from this "concept".
